config/initializers/apipie.rb

```diff
@@ -27,6 +27,8 @@
     trans
   end
   config.help_layout = 'apipie_dsl/apipie_dsls/help.html.erb'
+  config.default_class_description = ->(model) { _("A class representing %s object") % model.human }
```

Comment:
@ofedoren I recall you saying we need a bug fix in apipie-dsl to make this work

Reply:
It works, but just to be clear: if we want to defer/postpone translation until it's actually needed (on docs demand) instead of trying to translate on initialization (class parsing), then yes, we need a small fix in apipie-dsl lib.